### ISSUE TITLE
overflow-y of body is not removed on lightbox close

### ISSUE THREAD

**Issue by NickHatBoecker** [XiongAmao/vue-easy-lightbox#110]
I'm using the version for Vue2. When closing the lightbox via ESC, close button or mask the `body` element stil has `overflow-y: hidden` set.

Currently I'm using a workaround to fix this.
Also `index` isn't reactive. I have to use `@on-index-change` to update current index variable.

```vuejs
<template>
    <vue-easy-lightbox
        moveDisabled
        scrollDisabled
        :visible="show"
        :imgs="images"
        :index="currentIndex"
        @on-index-change="onIndexChange"
        @on-error="close"
        @hide="close"
    >
        <template #toolbar>
            <span></span>
        </template>
    </vue-easy-lightbox>
<template>

<script>
import VueEasyLightbox from 'vue-easy-lightbox'

export default {
    components: { VueEasyLightbox },

    data: () => ({ currentIndex: 0, show: false }),

    mounted () {
        setTimeout(() => {
            // Timeout in order to load the first image
            this.show = true
        }, 2000)
    },

    computed: {
        images () {
            return this.$store.getters.images
        },
    },

    methods: {
        close () {
            document.querySelector('body').style['overflow-y'] = 'visible'
        },

        onIndexChange (oldIndex, newIndex) {
            this.currentIndex = newIndex
        },
    },
}
</script>
```

**Comment by XiongAmao** [comment]
Hi, 

Neither `index` nor `visible` is reactive. By design, you should control it by yourself. 
This means that you can change `visible` or index  at anytime to activate `vue-easy-lightbox` or switch images.

`overflow-y: hidden` will be clear when `visible` change back to `false`. 

Maybe you can provide a repo and I'll see what the issue is.

https://github.com/XiongAmao/vue-easy-lightbox/blob/f848e225d9c9d43dd5d52d543304a7f3bdf4c936/src/vue-easy-lightbox.vue#L556-L593

**Comment by aasutossh** [comment]
So I was using two `<vue-easy-lightbox>`es with same `:visible` option. 
After removing one of them, it worked.

**Comment by XiongAmao** [comment]
The issue was fixed in v1.11.0
